aspnetcore/performance/response-compression/sample/License.txt

```diff
@@ -0,0 +1,12 @@
+Copyright (c) .NET Foundation. All rights reserved.

```

Comment:
@Rick-Anderson What's the guidelines on licenses? I see some samples with and some samples without. Would be nice to be consistent.